### PR TITLE
Feature/resource tag rewrite

### PR DESCRIPTION
**Pull request by ezkat**
Better handling for resource keywords when added through RA.
Added new resource filter "?keywords=foo,bar" for more accurate filtering, alas resource "?search" filter still works the old way.

### CODE REVIEW

**Review by aceViilee — APPROVED** [review]
Comment more. What happens and more importantly why.

File: resources/admin/__init__.py [hunk 1]
```diff
@@ -99,22 +99,86 @@ class UnitIdentifierInline(admin.StackedInline):
...
+        (_('Extra Information'), {
+            'fields': (
+                'tags',
+                'external_reservation_url',
+                'id',
+            ),
+        }),
+    )
+
+    readonly_fields = ('tags', )
```
Comment (aceViilee) [inline resources/admin/__init__.py:181]: Read only?

Comment (ezkat) [reply resources/admin/__init__.py:181]: I set the resource model "tags" field to read-only https://github.com/codepointtku/respa/blob/977eb28cdb189d496bbae43a1854d95a458af707/resources/models/resource.py#L230 in favor of using the new resourcetag model https://github.com/codepointtku/respa/blob/977eb28cdb189d496bbae43a1854d95a458af707/resources/models/resource.py#L92 via the inline field https://github.com/codepointtku/respa/blob/6ce84afe11eed3228c77f9dfbceee436d577094b/resources/admin/__init__.py#L102

Comment (aceViilee) [reply resources/admin/__init__.py:181]: Ok. Good

**Review by hienous — APPROVED** [review]

**Pull request merged**